### ISSUE TITLE
Web interface with advanced poll features for icons and rendering proposals

### ISSUE THREAD

**Issue by althio** [hotosm/hotosm-project-ideas#19]
Proposed by: @althio

**Project (student):** Web interface with advanced poll features
**Internal objective (OSM-HOT):** Tool to aid decision making on rendering

**Summary:** Create a tool to review proposals, gather feedback and analyse results.

The tool would allow maintainer to propose one or several rendering for a particular tag. Then everyone could vote with a scale of satisfaction about the proposal. Satisfaction could be refined with several scales eg: aesthetics, understanding, technical requirements.

Profile information could be added by participants about their skills, age group, cultural background or knowledge about specific culture. If any, sensitive characteristics are supplied only by free choice. Poll may need to be anonymized anyway.

Poll results should be consolidated as overall score, table or graphical results. The analysis of poll results, weighted accordingly to participant profiles, would help for decision and triage of suitable proposals. A rendering could be marked as inappropriate for a specific reason (eg. cultural misunderstanding or unmet technical requirements) so that the need for improvement is clear.

The student project could certainly be restricted to a functioning flexible framework with mostly technical input. Any refinement about wording, ethics, legal issues could be handled in parallel or later.

**Skills Required:** web interface & ergonomics, database, data mining/analysis/profiling, information architecture, polls (with relative considerations such as sampling, statistical corrections for sampling bias, privacy and anonymous content).

**Possible Mentors:** @yohanboniface? @skorasaurus? @dalekunce?
Non-technical co-mentoring? @cgiovando, @mataharimhairi, @althio, @nebulon42?

Notes: related to project https://github.com/hotosm/HDM-CartoCSS/
Subset or related to #7.

**Comment by yohanboniface** [comment]
I don't think we need such a system. If we were in a situation were the Github issues were full of comments and debates, we'd certainly benefit from something like that, but that's not the situation. So better spend energy on more useful tasks imho.

**Comment by althio** [comment]
Hi @yohanboniface, thank you for the feedback.

I guess you are referring to the Github issues from HDM CartoCSS, aren't you?
I agree this is relatively quiet and my point of view is that it does need more input.

We were also talking about shared effort on icons, something potentially cross-project with HDM, openstreetmap-carto, Maki, Nori, maybe also country- or culture-dependant styles.

I feel the heated debate is sometimes in repo openstreetmap-carto whereas the other places are less exposed.

Also regardless of debate intensity all these Github repositories are accessible only to people with the correct combination of enough knowledge, time, technical skills and english profiency.

We need more diverse and cultural feedback https://github.com/hotosm/HOT-Project-Ideas/issues/7#issuecomment-74990868 so we could use a non-tech user-friendly graphics-only no-text interface.

HDM CartoCSS is also facing more pressing problems where you want to spend energy and I respect that.

I would still argue that the present idea is not urgent, yet certainly useful. Maybe it needs to be brought at OSM-level or maybe the HOT community can kickstart it.
